### ISSUE TITLE
get twitter_followers on many slugs at once

### ISSUE THREAD

**Issue by marc-moreaux** [santiment/sanpy#137]
Hello, 

I was searching for something similar to @phisanti for the same reasons as him.
It appears that your solution (which I find great) seems to not be working for the metric "twitter_followers"
would you have any insight on why ?

``` python
{
  getMetric(metric: "twitter_followers") {
    timeseriesDataPerSlug(
      selector: {slugs: [
        "ethereum",
        "bitcoin"
      ]},
      from: "utc_now-7d",
      to: "utc_now"
      interval: "1d"){
        datetime
        data {slug value}
      }
  }
}
```

Thank you very much :)

_Originally posted by @marc-moreaux in https://github.com/santiment/sanpy/issues/135#issuecomment-978172398_

**Comment by tspenov** [comment]
Yes `timeseriesDataPerSlug` is not implemented for `twitter_followers` metric. This metric has different internal implementation than most of our other metrics. We've started the migration and the new implementation so hopefully we will have it soon.
